Detectors/MUON/MCH/Simulation/CMakeLists.txt

```diff
@@ -38,7 +38,9 @@ o2_target_root_dictionary(MCHSimulation
                                   include/MCHSimulation/Hit.h
                                   include/MCHSimulation/Response.h)
 
-o2_data_file(COPY data DESTINATION Detectors/MCH/simulation)
+INSTALL(DIRECTORY data DESTINATION share/Detectors/MCH/simulation)
+				
+#o2_data_file(COPY data DESTINATION Detectors/MCH/simulation)
```

Comment:
this should work. It does for everyone else.

Reply:
yes, why the change from o2_data_file to INSTALL ? Was there a problem ?

Reply:
I just cross-checked. It's ok with `o2_data_file`.